sources/operator.1.10.0.pxc-operator.dep.json

```diff
@@ -1,14 +1,14 @@
 {
   "backup": {
-    "8.0.23": {
```

Comment:
Well it will return 8.0.25/5.7.35 backup image, but it was like that before. We always return the latest backup image - we have many PXC versions in version service but it was always returning latest backup image.